<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="stn_1">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>선택1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="stn_2">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>30</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>선택1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="stn_3">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>30</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>선택1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="stn_4">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>80</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>선택1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="stn_5">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>80</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>선택1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="stn_6">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>70</y>
     <width>80</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>선택1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="stn_7">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>130</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>선택1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="stn_8">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>130</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>선택1</string>
   </property>
  </widget>
  <widget class="QPushButton" name="stn_9">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>130</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>선택1</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
